model/vfs/couchdb_indexer.go

```diff
@@ -151,10 +147,7 @@ func (c *couchdbIndexer) prepareFileDoc(doc *FileDoc) error {
 	return nil
 }
 
-// DirSizeAndCount returns the size taken by the directory and the files inside
-// it (including in the children directories), and also the number of files
-// inside it.
-func (c *couchdbIndexer) DirSizeAndCount(doc *DirDoc) (int64, int64, error) {
+func (c *couchdbIndexer) DirSize(doc *DirDoc) (int64, error) {
```

Comment:
We don't need to document exposed functions anymore?

Reply:
It depends, but it is not mandatory when there is an interface that already documents the function